changelogs/fragments/167-openssl_csr-crl-distribution-points.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+- "openssl_csr, openssl_csr_pipe - allow to specify CRL distribution endpoints (https://github.com/ansible-collections/community.crypto/issues/147, https://github.com/ansible-collections/community.crypto/pull/167)."
```

Comment:
```suggestion
- "openssl_csr, openssl_csr_pipe - allow to specify CRL distribution endpoints with ``crl_distribution_points`` new option (https://github.com/ansible-collections/community.crypto/issues/147, https://github.com/ansible-collections/community.crypto/pull/167)."
or something like that to mention the option name
```

Reply:
Sorry for editing your suggestion, this way it's still your suggestion and not my own ;)